### PR TITLE
extend Replacement and Typo rules

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: src/Rule/Typo.php [hunk 1]
```diff
@@ -60,6 +60,8 @@ public static function getList(): array
             '/recalcuate/i' => null,
             '/achived/i' => null,
             '/overriden/i' => null,
+            '/succesfully/i' => null,
+            '/optionnally/i' => null,
```
Comment (xabbuh) [inline src/Rule/Typo.php]: Should we also add "successfull" and "helpfull"?

Comment (OskarStark) [reply src/Rule/Typo.php]: Yes, do you often face issues with misspelling here?

Comment (xabbuh) [reply src/Rule/Typo.php]: yes, I have seen that several times

Comment (OskarStark) [reply src/Rule/Typo.php]: @xabbuh Can you name some common misspellings I can match?

Comment (xabbuh) [reply src/Rule/Typo.php]: besides "helpfull" and "usefull" I don't remember anything else

Comment (OskarStark) [reply src/Rule/Typo.php]: ok

**Pull request merged**